lib/manager/components/FeedSourceTableRow.js

```diff
@@ -124,46 +119,69 @@ function relativeDuration (time: any) {
   )
 }
 
+// This method is extracted variable generation which had to be split
```

Comment:
Sentence is incomplete. Also use `/**`-style.

Reply:
"Refactored"!